pokedex_async_redux/lib/utils/string_constants.dart

```diff
@@ -0,0 +1,46 @@
+const offsetPokemon = '0';
+const limitPokemon = '151';
+
+///Pokemon Overview Page String Constants
+const imageUrl = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/[index].png';
+const indexKey = '[index]';
+const String pokemonOverviewTitle = 'POKEDÉX';
+const String type = 'Type';
+const String typePrimary = 'Type 1';
+
+///Pokemon Loading and Error Message
+const String pikachuRunningImage = 'assets/images/pikachu-running.gif';
```

Comment:
also here no need to indicate its String type, same goes for all in this entire file 🙂